src/main/kotlin/no/digdir/fdk/searchservice/mapper/ConceptMapper.kt

```diff
@@ -0,0 +1,23 @@
+package no.digdir.fdk.searchservice.mapper
+
+import no.digdir.fdk.searchservice.model.*
+
+fun Concept.toSearchObject() =
+    SearchObject(
+        id = id,
+        uri = "",
+        accessRights = null,
+        catalog = null,
+        dataTheme = null,
+        description = definition?.text,
+        fdkFormat = null,
+        harvest = null,
+        isOpenData = null,
+        keyword = null,
+        losTheme = null,
+        organization = publisher?.toOrganization(),
```

Comment:
```suggestion
        organization = publisher,
```

Reply:
Dette er exention-function som ligger på objektet Publisher for å mappe den til type Organization